cli/connect/common.go

```diff
@@ -0,0 +1,114 @@
+package connect
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/adam-hanna/arrayOperations"
+	"github.com/c-bata/go-prompt"
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+const (
+	MaxHistoryLines = 10000
+
+	TCPNetwork  = "tcp"
+	UnixNetwork = "unix"
+)
+
+type ConnOpts struct {
+	Network  string
+	Address  string
+	Username string
+	Password string
+}
+
+type GetRawSuggestionsFunc func(console *Console, lastWord string) interface{}
+
+func getConnOpts(connString string, ctx *context.Ctx) (*ConnOpts, error) {
+	connOpts := ConnOpts{
+		Username: ctx.Connect.Username,
+		Password: ctx.Connect.Password,
```

Comment:
\+ timeout?

Reply:
Timeout is a constant that doesn't depend on conn string.